wg-etcd-operator/annual-report-2024.md

```diff
@@ -2,26 +2,25 @@
 
 ## Current initiatives and Project Health
 
-
 1. What work did the WG do this year that should be highlighted?
 
-<!--
-   Some example items that might be worth highlighting:
-   - artifacts
-   - reports
-   - white papers
-   - work not tracked in KEPs
--->
+* We launched the WG, and [recruited contributors](https://github.com/etcd-io/etcd-operator/graphs/contributors) from multiple prior operators
+* Completed [review of leading existing operators](https://github.com/etcd-io/etcd-operator/blob/main/docs/wg/evaluation/evaluation.pdf)
+* [Surveyed potential users](https://github.com/etcd-io/etcd-operator/blob/main/docs/wg/survey/2024-operator-survey.md) and determined [operator development roadmap](https://github.com/etcd-io/etcd-operator/blob/main/docs/roadmap.md)
+* Created initial [operator workflow]()
```

Comment:
Once https://github.com/etcd-io/etcd-operator/pull/57 gets merged, then you can update link here
```suggestion
* Created initial [operator reconciliation workflow]()
```

Reply:
Link to workflow: https://github.com/etcd-io/etcd-operator/blob/main/docs/design/reconcile_loop_v0.1.0.png